tests/modules/ebi-metagenomics/foldcomp/decompress/test.yml

```diff
@@ -0,0 +1,27 @@
+- name: foldcomp decompress test_foldcomp_decompress
+  command: nextflow run ./tests/modules/ebi-metagenomics/foldcomp/decompress -entry test_foldcomp_decompress -c ./tests/config/nextflow.config
+  tags:
+    - foldcomp
+    - foldcomp/decompress
+  files:
+    - path: output/foldcomp/test_fcz/4590051235.fcz
```

Comment:
aren't the .`fcz` files the input? How come they are part of the output in your test?

Reply:
(please ignore this if I've misunderstood what the tool generates)

Reply:
Chained modules generate checks for the first module outputs as well with the automatic pytest building.